src/ensembl/src/content/app/help/Help.tsx

```diff
@@ -31,28 +31,36 @@ import {
 import { Menu as MenuType } from 'src/shared/types/help-and-docs/menu';
 import {
   TextArticleData,
-  VideoArticleData,
-  IndexArticleData
+  VideoArticleData
 } from 'src/shared/types/help-and-docs/article';
 
 import styles from './Help.scss';
 
-type ArticleData = TextArticleData | VideoArticleData | IndexArticleData;
+type ArticleData = TextArticleData | VideoArticleData;
 
 const Help = () => {
   const location = useLocation();
+  const isIndexPage = isIndexRoute(location.pathname);
   const { data: menu } = useApiService<MenuType>({
     endpoint: `/api/docs/menus?name=help`
   });
   const { data: article } = useApiService<any>({
-    endpoint: `/api/docs/article?url=${encodeURIComponent(location.pathname)}`
+    endpoint: `/api/docs/article?url=${encodeURIComponent(location.pathname)}`,
+    skip: isIndexPage
   });
+  const main = isIndexPage ? (
+    <main className={styles.main}>
+      <HelpLanding />
+    </main>
+  ) : article ? (
+    <MainContent article={article} />
+  ) : null;
```

Comment:
Do we need the loading spinner here?

Reply:
The reason I added a loading spinner to the `About` component was that without it, the layout of the page was breaking while an article was loading: the content of the right column got shifted to the empty space to the left. With help pages, we don't have this problem — while an article is being loaded, the relevant part of the page just remains uniformly blank. I think we can add the spinner later if we want to.